inst/templates/webpack.config.js.txt

```diff
@@ -1,15 +1,17 @@
 var path = require('path');
 
 module.exports = {
-    entry: path.join(__dirname, 'srcjs', '${name}.js'),
+    mode: 'development',
+    entry: path.join(__dirname, 'srcjs', '${name}.jsx'),
```

Comment:
@alandipert  `scaffoldReactWidget` no longer works since webpack now looking for `.jsx`  while `scaffoldReactWidget` creates `{name}.js` [line](https://github.com/react-R/reactR/pull/22/files#diff-8fce518b699dc1cdd6684eb836823c15R80).  We get error

```
$ C:\Users\kentr\Dropbox\dev2\r\tester\node_modules\.bin\webpack --mode=development

Insufficient number of arguments or no entry found.
Alternatively, run 'webpack(-cli) --help' for usage info.

Hash: c2febaaed3dac7951d52
Version: webpack 4.29.6
Time: 114ms
Built at: 2019-03-02 09:06:08

ERROR in Entry module not found: Error: Can't resolve 'C:\Users\kentr\Dropbox\dev2\r\tester\srcjs\tester.jsx' in 'C:\Us
ers\kentr\Dropbox\dev2\r\tester'
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

I guess we could either just change extension with `scaffoldReactWidget` to `.jsx` even though we don't plan to use `JSX` syntax, or we should have a different `webpack.config.js` for `scaffoldReactWidget`.

Reply:
@alandipert works for me